README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://api.travis-ci.org/locationtech/geotrellis.svg?branch=master)](http://travis-ci.org/locationtech/geotrellis) [![Join the chat at https://gitter.im/geotrellis/geotrellis](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/geotrellis/geotrellis?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Maven Central](https://img.shields.io/maven-metadata/v/http/central.maven.org/maven2/org/locationtech/geotrellis/geotrellis-spark_2.11/maven-metadata.xml.svg)](http://search.maven.org/#search%7Cga%7C1%7Corg.locationtech.geotrellis)
 [![ReadTheDocs](https://readthedocs.org/projects/geotrellis/badge/?version=latest)](http://geotrellis.readthedocs.io/en/latest/)
-[![Changelog](https://img.shields.io/badge/changelog-v1.2.0-brightgreen.svg)](https://geotrellis.readthedocs.io/en/latest/CHANGELOG.html)
+[![Changelog](https://img.shields.io/badge/changelog-v1.2.0-brightgreen.svg)](https://github.com/locationtech/geotrellis/blob/master/CHANGELOG.md)
```

Comment:
I rebuilt the docs after the changelog move in this PR by executing:
```
pushd docs
make clean && make html && make serve
```
and then viewing the updated docs at localhost:8000. There's some weirdness in that the Changelog link remains on some pages, but not others. Since we're moving the changelog now, we should do some cleanup here. If we don't want the changelog displayed on the read the docs site at all, we should ensure links to it are entirely removed. If we do still want it displayed, we should make sure the link is consistently available and it renders from markdown correctly.

<img width="351" alt="Screen Shot 2019-10-22 at 8 57 17 AM" src="https://user-images.githubusercontent.com/1818302/67290963-d239c900-f4ae-11e9-9243-4732fdbdb7f4.png">
<img width="444" alt="Screen Shot 2019-10-22 at 8 57 28 AM" src="https://user-images.githubusercontent.com/1818302/67290965-d239c900-f4ae-11e9-854c-796edaf2a4a4.png">

Reply:
could not reproduce this one but I manually deleted the `_build` folder.
Why `Create a Contribution Questionnaire` gets promoted to the side panel and is absent from current page I can not say.

Reply:
actually this could have been a strange leftover from TOC, took care of that

Reply:
> manually deleted the _build folder.

Huh. So did I, or at least that's what `make clean` claims it did. Oh well, good enough for now, agree that cleaning this up further should be a separate task.